book/glossary/glossary.tex

```diff
@@ -0,0 +1,374 @@
+A strong dough can be stretched without tearing and holds its shape well. This is
+largely influenced by the flour's protein content and the development of the gluten
+network.
+
+\textbf{Dutch oven:} A heavy-duty pot with a tight-fitting lid, often made of cast
+iron. It's used in baking to trap steam during the initial phase of baking, helping
+to create a crusty exterior on bread.
+
+\textbf{Feed:} The act of adding fresh flour and water to maintain a sourdough
+starter. Regular feeding keeps the starter active and healthy.
+
+\textbf{Fermentation:} The metabolic process by which microorganisms such as yeast
+and bacteria convert carbohydrates (like sugars) into alcohol or acids. In bread
+making, this produces carbon dioxide which causes the dough to rise.
+
+\textbf{Fermentolyse:} Using a small amount of starter to slow fermentation.
+It's a method where fermentation and autolyse are combined. Typically around 10\%
+of starter is used for the fermentolyse. The flour, water and starter are mixed
+together. By adding the starter early the dough becomes more extensible and easier
+to handle.
+
+\textbf{Fool’s Crumb:} A term used to describe a crumb structure that has several
+large pockets or holes, rather than an even distribution of smaller holes. This
+isn't necessarily a desired feature, as it can indicate uneven fermentation or
+improper shaping techniques.
+
+\textbf{Elasticity:} A property of dough that describes its ability to return to
+its original shape after being stretched or deformed. It's influenced by the flour's
+protein content and the development of the gluten network.
+
+\textbf{Extensibility:} Refers to the dough’s ability to be stretched or extended
+without tearing. It's the opposite of elasticity and is desirable in certain types
+of breads, like ciabatta, that have a more open crumb structure.
+
+\textbf{Homogenizing:} The act of creating a consistent and uniform mixture. For
+flours like einkorn and rye, where gluten alignment isn't the main goal, kneading
+ensures that the dough achieves this homogeneous consistency.
+
+\textbf{Hooch:} A liquid layer that sometimes forms on top of a sourdough starter.
+It's an indication that the starter is hungry and needs feeding. It acts as a
+barrier shield and prevents the starter from catching mold. It can be mixed right
+back into the starter or extracted to make hot sauces.
+
+\textbf{Gluten:} A protein complex formed from gliadin and glutenin, found in wheat
+and some other grains. It provides elasticity and strength to the dough when
+properly aligned and developed. During the course of the bulk fermentation much of
+the gluten is degraded by the protease enzyme and lactic acid bacteria.
+
+\textbf{Kneading:} The manual or mechanical process of working dough to develop gluten
+in wheat and spelt-based breads, or to homogenize the dough mass in flours like
+einkorn or rye.
+
+\textbf{Kochstück:} See \#Brühstück
```

Comment:
When making a Kochstück, the flour or grains are heated together with the fluid. The mixture needs to be stirred while heating up to prevent clumping and burning it.